### ISSUE TITLE
Variables are not matched over different GRAPH scopes

### ISSUE THREAD

**Issue by brechtvdv** [comunica/comunica#350]
#### Issue type:

- :bug: Bug

____
#### Description:

When running a SPARQL query against a file with a named graph and a TPF-interface with a default graph, following query doesn't give a result:
`node packages/actor-init-sparql/bin/query.js  -l debug file@https://gist.githack.com/brechtvdv/4b795e71035bac48ead93677d657d87e/raw/6586d4d74b97b168addb37f3699f189517134bc1/archief.jsonld hypermedia@https://query.wikidata.org/bigdata/ldf 'QUERY'`

With QUERY: 
```
SELECT * WHERE { 
	GRAPH ?g { 
		?krant <http://schema.org/mentions> ?person . 
	} 
	?person <http://www.wikidata.org/prop/direct/P569> ?o . 
}
```

This query does work:
```
SELECT * WHERE {
	GRAPH ?g {
		?krant <http://schema.org/mentions> <http://www.wikidata.org/entity/Q352> . 
	}
	<http://www.wikidata.org/entity/Q352> <http://www.wikidata.org/prop/direct/P569> ?o . 
}
```

**Comment by brechtvdv** [comment]
Is this a lot of work to fix?
I would need this in 2 weeks for a project

**Comment by rubensworks** [comment]
@brechtvdv The problem may be related to bnodes in the data itself, like here: https://github.com/comunica/comunica/issues/343

**Comment by rubensworks** [comment]
@brechtvdv I assume this issue can be can be closed because #343 has been fixed? Or are there any other issues related to this?

**Comment by brechtvdv** [comment]
@rubensworks Yes, this can be closed :)
